kato/kato-azure/kato-azure.gradle

```diff
@@ -1,7 +0,0 @@
-dependencies {
-  def azureSDKVersion = '0.8.2'
```

Comment:
Where did this come from? It's not even close to the right version number. Where is it being used?